chromium/storage.js

```diff
@@ -1,6 +1,10 @@
 var storage = chrome.storage.local;
```

Comment:
Within for example the tor browser, `chrome.storage.sync` may exist but throw an error.  That's the reason for the `lastError` check.